timemachine/fe/bar.py

```diff
@@ -287,12 +291,12 @@ def pair_overlap_from_ukln(u_kln: NDArray) -> float:
     Returns
     -------
     pair_overlap: float
-        2 * pymbar.MBAR overlap
+        2 * pymbar.mbar.MBAR overlap
         (normalized to interval [0,1] rather than [0,0.5])
 
     """
     u_kn, N_k = ukln_to_ukn(u_kln)
-    return 2 * pymbar.MBAR(u_kn, N_k).computeOverlap()["matrix"][0, 1]  # type: ignore
+    return 2 * pymbar.mbar.MBAR(u_kn, N_k).compute_overlap()["matrix"][0, 1]  # type: ignore
```

Comment:
nit: tangential to this PR, but while here we might replace `2 * compute_overlap()["matrix"][0, 1]` with `compute_overlap()["scalar"]`. (I believe they are equivalent, but would double-check.)

Reply:
Updated in https://github.com/proteneer/timemachine/pull/1492/commits/161d7c22374f67b1386c4b75a9984c4ff12ee1cc